app/src/components/ChoiceBoard/useChoiceBoardModel.tsx

```diff
@@ -0,0 +1,31 @@
+import {useState} from 'react';
+
+export default (originalOptions) => {
+    // Selected Items
+    const [selectedItems, setSelectedItems] = useState(Array());
+
+    // All choices
+    const [allOptions, setAllOptions] = useState(originalOptions);
+
+    // Progress
+    // const [progress, setProgress] = useState(0);
+
+    const appendSelection = (selection) => {
+        setSelectedItems([...selectedItems, selection]);
+
+        // Mock POST, not sure if we are going to use axios or fetch
+        // This function will not only append the current choice to the list of choices, 
+        // but also return the two new choices
+        // axios.post('http://localhost:8008/items', {selection})
```

Comment:
I know this code is commented, but, answearing your question in comments, we will use `fetch` with async/await

Reply:
Tiago, 

I will mock up the fetch and async/await when I implement the other suggested changes.

Thanks,

Reply:
Awesome!